### PR TITLE
Fix middle area's bug in generating Star shape

### PR DESCRIPTION
**Pull request by csyuhao**
Ref: #1

### CODE REVIEW

File: generator/shapes.py
Comment (elkorchi) [inline generator/shapes.py]: Can you please reformat this line to comply with PEP8 standard (100 characters per line)

File: generator/shapes.py
Comment (elkorchi) [inline generator/shapes.py]: The abstract shape is meant to be an abstract class that does not include any specific logic of other shapes, is it possible to include those updates inside the Star class, I suggest to override the full draw method

**Comment by elkorchi** [comment]
@yuhaoooo Thank you for the PR, I have some small comments for your PR, once addressed should be good to merge, If you don't have time to do it please ping and I will do it

**Comment by csyuhao** [comment]
What comments?

**Comment by elkorchi** [comment]
> What comments?

**Comment by elkorchi** [comment]
> What comments?

@yuhaoooo is my comments not visible for you ?

**Comment by csyuhao** [comment]
I have update this file.

**Review by elkorchi — APPROVED** [review]

**Comment by elkorchi** [comment]
Merged, Thanks @yuhaoooo for your contribution

**Pull request merged**
